codec/src/main/java/io/netty5/handler/codec/DelimiterBasedFrameDecoder.java

```diff
@@ -113,9 +113,9 @@ public DelimiterBasedFrameDecoder(
      */
     public DelimiterBasedFrameDecoder(
             int maxFrameLength, boolean stripDelimiter, boolean failFast,
-            ByteBuf delimiter) {
-        this(maxFrameLength, stripDelimiter, failFast, new ByteBuf[] {
-                delimiter.slice(delimiter.readerIndex(), delimiter.readableBytes())});
+            Buffer delimiter) {
```

Comment:
@chrisvest What do you think about removing the constructor

```
    public DelimiterBasedFrameDecoder(
            int maxFrameLength, boolean stripDelimiter, boolean failFast,
            Buffer delimiter) {
```

And keep only this one

```
    public DelimiterBasedFrameDecoder(
            int maxFrameLength, boolean stripDelimiter, boolean failFast, Buffer... delimiters) {
```

Reply:
@violetagg sounds good